app/views/assessment/index.js

```diff
@@ -127,84 +139,66 @@ const Assessment = ({ navigation }) => {
             screenOptions={{
               cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS,
               cardStyle: {
-                backgroundColor: 'transparent',
+                backgroundColor: Colors.transparent,
               },
             }}>
             <Stack.Screen
               component={AssessmentStart}
               name='AssessmentStart'
               options={{
-                ...screenOptions,
+                ...screenOptions(),
+                headerLeft: () => null,
                 headerRight: () => null,
-                headerStyle: {
-                  backgroundColor: Colors.primaryBackgroundFaintShade,
-                },
               }}
             />
+            <Stack.Screen
```

Comment:
Would it makes sense to have the navigator for the self assessment screen live with the other navigators? Or is there a good reason to keep it separate?

Reply:
Hmm, I am not sure yet because the file contains a lot of logic specific for the assessment. I do think we should be able to do this once we step away from the wey question flow is handled right now, and remove some of that logic.